### DISCUSSION TITLE
Get data of logged in user

### DISCUSSION THREAD

**Discussion by Akshaykakkodi** [pocketbase/pocketbase#4751] [Q&A]
How to get the data of user logged in from users collection, apart from the respose we get while logging in

**Comment by ganigeorgiev** [comment]
It is not clear what actually you are asking.

Based on your other [recent question](https://github.com/pocketbase/pocketbase/discussions/4678) I assume that you are asking how to get a user response using the Dart SDK. In that case you can simply send `pb.collection("users").getOne(pb.authStore.model.id)` request.

You can also find more details in the "API preview" section for your collection in the Admin UI or check the general docs in https://pocketbase.io/docs/api-records/.

If you need further help, please don't just prompt and try to elaborate as much as possible on your use case to understand what you are trying to do.
